main.tf

```
@@ -163,8 +163,8 @@ resource "azurerm_managed_disk" "this" {
   create_option        = "Empty"
   disk_size_gb         = each.value.disk_size_gb
   storage_account_type = each.value.storage_account_type
-
-  tags = var.tags
+  hyper_v_generation   = each.value.hyper_v_generation
+  tags                 = var.tags
```

Restore intentional whitespace:

```suggestion
  hyper_v_generation   = each.value.hyper_v_generation
  
  tags = var.tags
```